README.md

```diff
@@ -164,16 +165,8 @@ After publishing:
 If you get the error message `release version 17 not supported`:
 * Run `java -version` and verify that you're running 17. If not, you will need to install / update your PATH.
 
-If you have trouble submitting workflows and see errors like `HTTP error calling URI https://cromiam-priv.dsde-dev.broadinstitute.org`:
-* Connect to the NonSplit VPN and try again
-* CromIAM doesn't accept requests from outside the Broad trusted IP space 
-
-When running back Rawls with a DB clone, the app may crash on launch with an error related to `OpenTelemetry`.
-* Work around by setting `entityStatisticsCache.enabled = false`.
```

Comment:
is this not a problem anymore?

Reply:
I originally added this comment last year and I didn't run into the issue re-testing the same scenario recently.